source/css/sass/mixins/_spacing.scss

```diff
@@ -188,151 +188,32 @@
   @include padding(0, $dimension);
 }
 
-// blg stands for "baseline grid"
-
-@mixin blg-spacing($dimension, $level, $type: "padding") {
-  @if $type == "padding" {
-    @if $level == "extra-small" {
-      @include padding($baselinegrid-space-extra-small, $dimension);
-    } @else if $level == "small" {
-      @include padding($baselinegrid-space-small, $dimension);
-    } @else if $level == "smallish" {
-      @include padding($baselinegrid-space-smallish, $dimension);
-    } @else if $level == "medium" {
-      @include padding($baselinegrid-space-medium, $dimension);
-    } @else if $level == "large" {
-      @include padding($baselinegrid-space-large, $dimension);
-    }
-  } @else if $type == "margin" {
-    @if $level == "extra-small" {
-      @include margin($baselinegrid-space-extra-small, $dimension);
-    } @else if $level == "small" {
-      @include margin($baselinegrid-space-small, $dimension);
-    } @else if $level == "smallish" {
-      @include margin($baselinegrid-space-smallish, $dimension);
-    } @else if $level == "medium" {
-      @include margin($baselinegrid-space-medium, $dimension);
-    } @else if $level == "large" {
-      @include margin($baselinegrid-space-large, $dimension);
-    }
-  }
-}
-
-@mixin blg-pad-top--small-to-medium {
-
-  @include blg-spacing("block-start", "small");
-
-  @include mq($from: medium) {
-    @include blg-spacing("block-start", "medium");
-  }
-
-}
-
-@mixin blg-pad-bottom--small-to-medium {
-
-  @include blg-spacing("block-end", "small");
-
-  @include mq($from: medium) {
-    @include blg-spacing("block-end", "medium");
-  }
-
-}
-
-@mixin blg-pad-top--large-to-extra-large {
-
-  @include blg-spacing("block-start", "large");
-
-  @include mq($from: medium) {
-    @include blg-spacing("block-start", "extra-large");
-  }
-
-}
-
-@mixin blg-pad-bottom--large-to-extra-large {
-
-  @include blg-spacing("block-end", "large");
-
-  @include mq($from: medium) {
-    @include blg-spacing("block-end", "extra-large");
-  }
-
-}
-
-@mixin blg-pad-vertical-small-to-medium {
-  @include blg-pad-top--small-to-medium();
-  @include blg-pad-bottom--small-to-medium();
-}
-
-@mixin blg-pad-vertical-large-to-extra-large {
-  @include blg-pad-top--large-to-extra-large();
-  @include blg-pad-bottom--large-to-extra-large();
-}
-
-@mixin blg-margin-bottom--medium-to-large {
-
-  @include blg-spacing("block-end", "medium", "margin");
-
-  @include mq($from: medium) {
-    @include blg-spacing("block-end", "large", "margin");
-  }
-
-}
-
-@mixin blg-margin-bottom--small-to-medium {
-
-  @include blg-spacing("block-end", "small", "margin");
-
-  @include mq($from: medium) {
-    @include blg-spacing("block-end", "medium", "margin");
-  }
-
-}
-
-@mixin blg-column-container() {
-  @include blg-margin-bottom--medium-to-large();
+@mixin block-spacing($end: $baselinegrid-space-small, $start: 0) {
+  @include margin($start $end, block);
```

Comment:
We should be consistent with usage of `block` or `"block"`.

Reply:
Generally we aren't using `"` to wrap strings in de novo Libero code, so lets make it official and say that we don't `"`-wrap strings in scss unless it's required by the composition of the string.

Short answer: `block`.